src/App.js

```diff
@@ -1,52 +1,85 @@
 import React from "react";
-import logo from "./logo.svg";
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch,
+  Redirect,
+} from "react-router-dom";
 import "./App.css";
-import FireStore from "./firebase/fireStore";
+import { getAuth } from "./utils";
+import Home from "./containers/Home";
+import About from "./containers/About";
+import Contact from "./containers/Contact";
+import Search from "./containers/Search";
+import Faq from "./containers/Faq";
+
+const ProcessedRoute = ({
+  component: Component,
+  type,
+  permissions,
+  redirectTo,
+  ...rest
+}) => {
+  const isLoggedIn = getAuth().isLoggedIn;
```

Comment:
@GeekySankar i wanted to sync w/ you to cover few aspects of design. Will do that tomorrow.
- Auth is not required
- Faq and Contact tabs are obe, not required;